tests/client/core/containers/TestServerHtml.js

```diff
@@ -0,0 +1,127 @@
+import Helmet from 'react-helmet';
+import React, { PropTypes } from 'react';
+
+import { findRenderedDOMComponentWithTag, renderIntoDocument } from 'react-addons-test-utils';
+import ServerHtml from 'core/containers/ServerHtml';
+
+describe('<ServerHtml />', () => {
+  beforeEach(() => {
+    Helmet.canUseDOM = false;
+  });
+
+  const fakeStore = {
+    getState: () => {
+      return {foo: 'bar'};
+    },
+  };
+
+  const fakeAssets = {
+    styles: {
+      disco: '/bar/disco-blah.css',
+      search: '/search-blah.css',
+    },
+    javascript: {
+      disco: '/foo/disco-blah.js',
+      search: '/search-blah.js',
+    },
+  };
+
+  const fakeSRIData = {
+    'disco-blah.css': 'sha512-disco-css',
+    'search-blah.css': 'sha512-search-css',
+    'disco-blah.js': 'sha512-disco-js',
+    'search-blah.js': 'sha512-search-js',
+  };
+
+  class FakeApp extends React.Component {
+    static propTypes = {
+      children: PropTypes.node,
+    }
+
+    render() {
+      const { children } = this.props;
+      return (
+        <div>
+          <Helmet
+            defaultTitle="test title"
+            meta={[{name: 'description', content: 'test meta'}]}
+          />
+          {children}
+        </div>
+      );
+    }
+  }
+
+  function render({ appName = 'disco',
+                    env = 'production', sriData = fakeSRIData } = {}) {
+    return renderIntoDocument(<ServerHtml assets={fakeAssets}
+                                          component={<FakeApp />}
+                                          store={fakeStore}
+                                          sriData={sriData}
+                                          appName={appName} env={env} />);
+  }
+
+
+  it('renders css provided', () => {
+    const html = findRenderedDOMComponentWithTag(render(), 'html');
+    const styleSheet = html.querySelector('link[rel="stylesheet"]');
```

Comment:
This seems like it could pass if the data happened to line up right even if it was just rendering all of the assets since `querySelector` will only get the first `link` tag.